packages/create-crc-app/templates/package.json

```diff
@@ -0,0 +1,46 @@
+{
+    "name": "{{name}}",
+    "version": "0.0.1",
+    "engines": {
+        "node": ">=16.0.0",
+        "npm": ">=7.0.0"
+    },
+    "scripts": {
+        "dev": "fec dev",
+        "lint": "eslint src",
+        "patch:hosts": "fec patch-etc-hosts",
+        "test": "jest"
+    },
+    "dependencies": {
+        "@patternfly/react-core": "^4.181.1",
+        "@redhat-cloud-services/frontend-components": "^3.6.1",
+        "prop-types": "^15.8.1",
+        "react": "17.0.2",
+        "react-dom": "17.0.2",
+        "react-redux": "^7.2.6",
+        "react-router-dom": "5.2.0",
+        "redux": "^4.1.2"
```

Comment:
Maybe we could skip Redux as a default dependency, there are (better) alternatives teams could use

Reply:
You actually require redux to use the FEC MAIN component which I think was used by QE? I wanted to drop it too. @karelhala is that still the case?